processor/logstransformprocessor/processor_test.go

```diff
@@ -164,10 +158,10 @@ func TestLogsTransformProcessor(t *testing.T) {
 			wantLogData := generateLogData(tt.parsedMessages)
 			err = ltp.ConsumeLogs(context.Background(), sourceLogData)
 			require.NoError(t, err)
-			time.Sleep(200 * time.Millisecond)
-			logs := tln.AllLogs()
-			require.Len(t, logs, 1)
-			assert.NoError(t, plogtest.CompareLogs(wantLogData, logs[0]))
```

Comment:
Looks like https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/9761 fails in this line due to receiving a wrong log record. So I'm not sure this fix will help

Reply:
oh ok good point. Looking.

Reply:
Couldn't figure it out. Closing.